src/manifolds/VectorBundle.jl

```diff
@@ -1002,6 +1155,16 @@ function vector_transport_to!(
     return copyto!(M.fiber.manifold, Y, p, X)
 end
 
+@inline function Base.view(x::ArrayPartition, M::VectorBundle, s::Symbol)
+    if s === :point
+        return x.x[1]
+    elseif s === :vector
+        return x.x[2]
+    else
+        throw(DomainError(s, "unknown component $s on $M."))
+    end
+end
```

Comment:
```suggestion
    (s === :point) && return x.x[1]
    (s === :vector) && return x.x[2]
    throw(DomainError(s, "unknown component $s on $M."))
end
```

this is of course equivalent, I just find this nicer in style, or is there a benefit to use if?

Reply:
They are essentially equivalent, I just find the if-elseif variant a bit more intuitive but I'm fine with either.

Reply:
I find the other shorter and more readable, but I also do not have a strong opinion on this (we use my proposed style in quite a few places I think).